pkg/dataplane/deployment.go

```diff
@@ -368,7 +368,7 @@ func (d *Deployer) addCertMounts(
 func (d *Deployer) addServiceExtraMounts(
 	service dataplanev1.OpenStackDataPlaneService,
 ) (*dataplanev1.AnsibleEESpec, error) {
-	baseMountPath := path.Join(ConfigPaths, service.Name)
+	baseMountPath := path.Join(ConfigPaths, service.Spec.EDPMServiceType)
```

Comment:
EDPMServiceType is currenetly missing form the nova service so this will likely break things
can we correct that first

Reply:
by the way i understand that EDPMServiceType defaults to the service name but it kind fo defeats the reason for makeign this change if we are still relying on that defaulting.

Reply:
this is being adressed by https://github.com/openstack-k8s-operators/openstack-operator/pull/943 so ill consider that resolved but that also need to be cherry-picked
ill add ye both as reviewers on that